### PR TITLE
Updated comments and string literals with the regard to the new branding

### PR DESCRIPTION
**Pull request by k-stachowiak**

### CODE REVIEW

File: tls-client/README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Server certificate:
     cert. version     : 3
     serial number     : 65:7B:6D:8D:15:A5:B6:86:87:6B:5E:BC
     issuer name       : C=BE, O=GlobalSign nv-sa, CN=GlobalSign Organization Validation CA - SHA256 - G2
-    subject name      : C=GB, ST=Cambridgeshire, L=Cambridge, O=ARM Ltd, CN=*.mbed.com
```
Comment (Patater) [inline tls-client/README.md]: This probably shouldn't change, unless the actual certificate from os.mbed.com has the new branding.

File: tls-client/README.md [hunk 2]
```diff
@@ -91,4 +91,3 @@ This probably means you need to update the contents of the `SSL_CA_PEM` constant
 Another possible reason for this error is a proxy providing a different certificate. Proxies can be used in some network configurations or for performing man-in-the-middle attacks. If you choose to ignore this error and proceed with the connection anyway, you can change the definition of `UNSAFE` near the top of the file from 0 to 1.
 
 **Warning:** this removes all security against a possible active attacker, so use at your own risk or for debugging only!
-
```
Comment (Patater) [inline tls-client/README.md]: Make any whitespace changes all together in a separate commit before any other commits.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# mbed TLS Examples on mbedOS
```
Comment (Patater) [inline README.md]: The readme files have been updated as part of https://github.com/ARMmbed/mbed-os-example-tls/pull/125

Could you please rebase your work on top of that?

**Comment by k-stachowiak** [comment]
I will resubmit the relevant changes from a proper source and after rebasing to the changes from https://github.com/ARMmbed/mbed-os-example-tls/pull/125

**Pull request closed without merging**